hoomd_benchmarks/microbenchmark_custom_trigger.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2021 The Regents of the University of Michigan
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Hard sphere Monte Carlo benchmark."""
+
+import hoomd
+from . import common
+from .configuration.hard_sphere import make_hard_sphere_configuration
+
+
+class NeverTrigger(hoomd.trigger.Trigger):
+    """A trigger that always returns False."""
+
+    def __init__(self):
+        hoomd.trigger.Trigger.__init__(self)
+
+    def compute(self, timestep):
+        """Compute the value of the trigger."""
+        return False
+
+
+class MicrobenchmarkCustomTrigger(common.ComparativeBenchmark):
+    """Measure the overhead of evaluating a custom trigger.
+
+    See Also:
+        `common.ComparativeBenchmark`
+    """
+
+    def make_simulations(self):
+        """Make the Simulation objects."""
+        path = make_hard_sphere_configuration(N=self.N,
+                                              rho=self.rho,
+                                              dimensions=self.dimensions,
+                                              device=self.device,
+                                              verbose=self.verbose)
+
+        variant = hoomd.variant.Ramp(A=0, B=1, t_start=0, t_ramp=100)
+
+        sim0 = hoomd.Simulation(device=self.device, seed=100)
+        sim0.create_state_from_gsd(filename=str(path))
+        sim0.operations.updaters.clear()
+        sim0.operations.computes.clear()
+        sim0.operations.writers.clear()
+        sim0.operations.tuners.clear()
+
+        trigger0 = hoomd.trigger.Periodic(phase=1000000000, period=1000000000)
```

Comment:
```suggestion
        trigger0 = hoomd.trigger.Periodic(phase=1e9, period=1e9)
```
(I was counting zeros)

Reply:
This needs to be an integer type, not a float. I added separators to make the numbers more readable.